feature_repo/feature_store.yaml

```diff
@@ -0,0 +1,4 @@
+project: test_custom
+registry: data/registry.db
+provider: local
+online_store: custom.online.mysql.MySQLOnlineStore
```

Comment:
How would the user package up their own connector using Pypi? Currently it looks like you are referencing a connector using a local module folder, but if they installed `feast-custom-stores-demo` using `pip install feast-custom-stores-demo`, how would they reference the class?

Reply:
It would be the fully qualified class name as it existed in the package right now, so even if this package was installed in a different virtualenv this class name should be the same (I think)

Reply:
It's still not clear what this path would be though. What should they set `online_store` to be?